api/room_resource/schema.py

```diff
@@ -14,6 +14,67 @@ class Meta:
         model = ResourceModel
 
 
+class PaginatedResource(graphene.ObjectType):
+    pages = graphene.Int()
+    query_total = graphene.Int()
+    has_next = graphene.Boolean()
+    has_previous = graphene.Boolean()
+    resources = graphene.List(Resource)
+
+    def __init__(self, **kwargs):
+        self.page = kwargs.pop('page', None)
+        self.per_page = kwargs.pop('per_page', None)
+        self.query_total
+        self.pages
+
+    def resolve_resources(self, info):
+        page = self.page
+        per_page = self.per_page
+        query = Resource.get_query(info)
+        if not page:
+            return query.all()
+
+        if page:
+            if page < 1:
+                return GraphQLError("No page requested")
+
+            page = page - 1
+            self.query_total = query.count()
+            result = query.limit(per_page).offset(page*per_page)
+            # import pdb; pdb.set_trace()
```

Comment:
@mutoromike It seems we are not making use of the commented line

Reply:
Thanks, this has been fixed!